packages/admin-ui-codemod/README.md

```diff
@@ -27,4 +29,28 @@ Migrate from `Set` to `Stack`
 
 ```sh
 npx admin-ui-codemod set-to-stack <path>
-```
\ No newline at end of file
+```
+
+#### `radio-review`
+
+Update `Radio` & `RadioGroup` props to the latest specs (0.126.0+)
+
+```sh
+npx admin-ui-codemod radio-review <path>
+```
+
+#### `toggle-to-switch`
+
+Migrate `Toggle` to `Switch`. Update `Switch` props to the latest specs (0.128.0+)
+
+```sh
+npx admin-ui-codemod toggle-to-switch <path>
+```
+
+#### `tag-review`
+
+Update `Tag` props to the latest specs (0.131.0+)
```

Comment:
I'd recommend updating it to the latest version +1 minor before merging
```suggestion
Update `Tag` props to the latest specs (0.132.0+)
```

Reply:
Done